Log search updates (#16793)
newrelic/docs-website: src/content/docs/codestream/observability/log-search.mdx

No need to slow down your investigation by context switching between your IDE and your browser to search logs. CodeStream brings the New Relic log-search experience right into your IDE! Click on the <DoNotTranslate>**View Logs**</DoNotTranslate> entry for any service listed in the CodeStream pane, or click on the <DoNotTranslate>**View Logs**</DoNotTranslate> icon in CodeStream's global navigation.

> The dropdown list at the top of the page defaults to which ever service is currently selected in the tree view in the CodeStream pane, but you can select any service or entity to view its logs. If there are partitions associated with the selected entity you can also constrain your search to specific partitions.

Reviewer: ```suggestion
The dropdown list at the top of the page reflects the currently selected service in the CodeStream pane's tree view. However, you're free to choose any service or entity to view its logs.

If the chosen entity has partitions, you can further refine your search by selecting specific partitions from the dropdown.
```

<img
  title="Search logs"
  alt="A screenshot of log search results"
  src={codestreamLogSearchResults}
/>

> Click on the carat to the left of any log entry to expand it and view the details. Hover over any value to easily copy it.

Reviewer: ```suggestion
Expand log entries for detailed information by clicking the carat (^) next to them. Simply hover over any value to copy it.
```
